Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentTest.kt

```diff
@@ -0,0 +1,46 @@
+package de.rki.coronawarnapp.ui.main.home
+
+import androidx.fragment.app.testing.launchFragment
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import testhelpers.BaseUITest
+
+@RunWith(AndroidJUnit4::class)
+class HomeFragmentTest : BaseUITest() {
+
+    @MockK lateinit var viewModel: HomeFragmentViewModel
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this, relaxed = true)
```

Comment:
you can get rid of Annotation.init by using @RunWith(MockKJUnitRunner::class) if you want

PS: if it wouldn't be you I would have to fight for my techLead advising not to use Junit4 ;-)

Reply:
I'd leave it as it is for now, as it's the same pattern then that we use for unit-tests. Also helps if we need to change to different runners in the future.